logstash_async/handler.py

```diff
@@ -29,14 +29,16 @@ class AsynchronousLogstashHandler(Handler):
     :param ca_certs: The path to the file containing recognized CA certificates.
     :param enable Flag to enable log processing (default is True, disabling
                   might be handy for local testing, etc.)
+    :param ensure_ascii Will be  all non-ASCII characters in the output escaped with \uXXXX sequences
+                        (default is True)
     """
 
     _worker_thread = None
 
     # ----------------------------------------------------------------------
     def __init__(self, host, port, database_path, transport='logstash_async.transport.TcpTransport',
                  ssl_enable=False, ssl_verify=True, keyfile=None, certfile=None, ca_certs=None,
-                 enable=True):
+                 enable=True, ensure_ascii=True):
```

Comment:
This is a pure Formatter option, no need to pass it through the handler (I see that it might be easier as you don't have to setup your own formatter, still, I don't want to pass the Formatter options through the handler).